pkgs/default.nix

```diff
@@ -2,17 +2,18 @@
 
 { nixpkgs
 , system
-, arch ? "aarch64"
+, _nativeSystem ? null
 , nixOnDroidChannelURL ? null
 , nixpkgsChannelURL ? null
 , nixOnDroidFlakeURL ? null
 }:
 
 let
-  nixDirectory = callPackage ./nix-directory.nix { };
+  nativeSystem = if _nativeSystem == null then system else _nativeSystem;
```

Comment:
What's the difference between `system`. `_nativeSystem`, `targetSystem` and `pkgs.system`? Could you maybe add some comments in there to explain the intended use cases?

Reply:
(sorry for the delay) Scattered some comments around.